src/com/pentalog/pentastagiu/homework2/Membership.java

```diff
@@ -0,0 +1,7 @@
+package com.pentalog.pentastagiu.homework2;
+
+public enum Membership {
```

Comment:
ar putea fi pus si in Customer, pentru ca membershipul e strans legat de customer + si discountul ar putea fi pastrat aici intr-un field private, pentru a avea totul legat de membership centralizat: numele impreuna cu valoarea asociata a discountului, ceea ce va face usor de modificat codul in viitor